test/suites/ACID.m

```diff
@@ -2434,6 +2435,26 @@
     set(ax4, 'Color', 'none');
 end
 % =========================================================================
+function [stat] = histogramPlot()
+
+  if ~exist('histogram')
```

Comment:
I was borrowing the code and I would personally go with `isHG2()` (which is implicitly Matlab + version check) since histogram has just been introduced in R2014b.